toml/src/main/kotlin/org/rust/toml/completion/LocalCargoTomlDependencyCompletionProvider.kt

```diff
@@ -58,7 +63,12 @@ class LocalCargoTomlDependencyCompletionProvider : TomlKeyValueCompletionProvide
 
         val indexService = CratesLocalIndexService.getInstance()
 
-        val versions = indexService.getCrate(name)?.sortedVersions ?: return
+        val versions = try {
```

Comment:
Let's extract `sortedVersions` from the try-catch
```Kotlin
val crate = try {
    cratesService.getCrate(name)
} catch (e: CratesIndexNotReadyException) {
    return
}
val versions = crate?.sortedVersions ?: return
```
and drop whitespace before this line for consistency with `completeKey`